File: 1627-last-moment-before-all-ants-fall-out-of-a-plank/1627-last-moment-before-all-ants-fall-out-of-a-plank.cs
public class Solution {
    public int GetLastMoment(int n, int[] left, int[] right) {
        int t = 0;
        if (left.Length > 0) {
            t = left.Max();
        }
        if (right.Length > 0) {
            t = Math.Max(t, n - right.Min());
        }
        return t;
    }
}

File: 1627-last-moment-before-all-ants-fall-out-of-a-plank/1627-last-moment-before-all-ants-fall-out-of-a-plank.java
class Solution {
    public int getLastMoment(int n, int[] left, int[] right) {
        int t = 0;
        if (left.length > 0) {
            t = Arrays.stream(left).max().getAsInt();
        }
        if (right.length > 0) {
            t = Math.max(t, n - Arrays.stream(right).min().getAsInt());
        }
        return t;
    }
}
